content/en/docs/TrueNAS/ssh-connections.md

```diff
@@ -0,0 +1,172 @@
+Test the configuration from another system by running the *sftp*, *ssh*, and *scp* commands as the user.
+*sftp* and *scp* will work but *ssh* will fail.
+
+## Creating an SSH Keypair
+
+TrueNAS generates and stores [RSA-encrypted](https://en.wikipedia.org/wiki/RSA_%28cryptosystem%29) SSH public and private keypairs in **System > SSH Keypairs**.
+These are generally used when configuring SSH Connections or SFTP Cloud Credentials.
+Encrypted keypairs or keypairs with passphrases are not supported.
+
+Keypairs can be automatically generated as needed when creating new SSH Connections or Replication tasks.
+To manually generate a new keypair, click **ADD** and give the keypair a unique name.
+
+<img src="/images/system-sshkeypairs-add.png">
+<br><br>
+
+Clicking the button to generate a keypair adds values to the public and private key fields.
+These values can be highlighted and copied or downloaded as text files for later use.
+
+## Creating a Semi-Automatic Connection Between Two TrueNAS Systems
+
+TrueNAS offers a semi-automatic setup mode that simplifies setting up an SSH connection with another FreeNAS or TrueNAS system without having to log in to that system to transfer SSH keys.
+This requires an SSH keypair on the local system and administrator account credentials for the remote TrueNAS.
+The remote system must also be configured to allow root access with SSH.
+The keypair can be generated as part of the semiautomatic configuration or manually created in **System > SSH Keypairs**.
+
+Go to **System > SSH Connections** and click **ADD**.
+
+<img src="/images/system-sshconnections-semiauto.png">
+<br><br>
+
+Be sure to use a valid URL scheme for the remote TrueNAS URL.
+Leave the username as *root* and enter the account password for the remote TrueNAS system.
+The private key can be imported from a previously created SSH keypair or created with a new SSH keypair.
+
+Saving the new configuration automatically opens a connection to the remote TrueNAS and exchanges SSH keys.
+
+## Creating a Manual Connection Between TrueNAS and a Second NAS System
+
+Choosing to manually set up the SSH connection requires copying a public encryption key from the local to remote system.
+This allows a secure connection without a password prompt.
+
+Start by generating a new SSH keypair in **System > SSH Keypairs**.
+Copy or download the value for the public key.
+The public key needs to be added to the remote NAS.
+When the remote NAS is not a TrueNAS system, please see the documentation for that system for instructions on adding a public SSH key.
+
+### Adding a Public SSH Key to the TrueNAS Root Account
+
+Log in to the TrueNAS system that generated the SSH keypair and go to **System > SSH Keypairs**.
+Open the keypair to use for the SSH connection and copy the text of the public SSH key or download the public key as a text file.
+
+<img src="/images/system-sshkeypairs-copypublic.png">
+<br><br>
+
+Log in to the TrueNAS system and go to **Accounts > Users**.
```

Comment:
May need to clarify which TrueNAS system here. I'm assuming it's the TrueNAS that didn't generate the SSH keypari?